### DISCUSSION TITLE
A repo with releases and lastversion not finding lastest version

### DISCUSSION THREAD

**Discussion by dkebler** [dvershinin/lastversion#116] [Q&A] [answered]
lastversion https://github.com/tasmota/install
CRITICAL: No release was found

https://github.com/tasmota/install/releases/tag/2023-10-19.r.ff3e3e9360d9f6702c02e0034110b0910a52ec2d

**Comment by dvershinin** [comment, marked as answer]
None of these are filed as stable, all being prereleases you have to pass the `--pre` switch. Even then it won't work as you want, I guess, because all the latest "versions" are nothing but references to commits. To me, it looks like they are misusing release tags for tracking upstream commits, without providing a clear version string... probably won't be supported.
